lib/ansiblelint/__main__.py

```diff
@@ -35,6 +35,14 @@
 import os
 
 
+__FORMATTERS = (formatters.Formatter,
+                formatters.QuietFormatter,
+                formatters.ParseableFormatter,
+                formatters.JsonFormatter,
+                formatters.JUnitXmlFormatter)
+_FORMATTERS = dict((fmtr.name(), fmtr) for fmtr in __FORMATTERS)
```

Comment:
plz use a dict comprehension instead of a dict constructor combined with a generator expression